asistente_ladm_col/logic/quality_rules/qr_restriction_duplicate_records.py

```diff
@@ -0,0 +1,97 @@
+"""
+/***************************************************************************
+                              Asistente LADM_COL
+                             --------------------
+        begin                : 2022-06-15
+        git sha              : :%H$
+        copyright            : (C) 2022 by Sergio Ramírez (BSF Swissphoto)
+        email                : seralra96@gmail.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames, LayerConfig
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4021,
+                                                           QRE_IGACR4021E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRDuplicateRestrictionRecords(AbstractLogicQualityRule):

```

Comment:
Please rename the file to `qr_restriction_duplicate_records.py` by `qr_duplicate_restriction_records.py`